app/internal/mail.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+from fastapi.templating import Jinja2Templates
+from fastapi_mail import ConnectionConfig, FastMail, MessageSchema
+from pydantic import EmailStr
+from pydantic.errors import EmailError
+
+import config
+
+templates = Jinja2Templates(directory=os.path.join("app", "templates"))
+
+# application name
+CALENDAR_SITE_NAME = "Calendar"
+# link to the home page of the application
+CALENDAR_HOME_PAGE = "calendar.com"
```

Comment:
The correct path is 'calendar.pythonic.guru', nevertheless I reckon app.url_path_for()

Reply:
I will change the name, thanks.
but the app.url_path_for will not work for two reasons:
look here: https://stackoverflow.com/questions/63682956/fastapi-retrieve-url-from-view-name-route-name
the results will be "/" or "/registration" which will not be good inside an email.
the other reason is that the internal part of the code doesn't use the app, or at least that how I used it

I will try to think of a better way to do this